src/assets/scripts/client/game/GameController.js

```diff
@@ -222,6 +243,7 @@ class GameController {
         this.game.score += GAME_EVENTS_POINT_VALUES[gameEvent];
 
         this.game_updateScore();
+        this.game_updateScoreHistory(GAME_EVENTS_DESCRIPTION[gameEvent], GAME_EVENTS_POINT_VALUES[gameEvent]);
```

Comment:
This is a great start! I'd encourage you to send only `gameEvent` here and have `game_updateScoreHistory` do the work of figuring out the event description and point value